test/specs/validate-spec/invalid/3.0/component-schema-with-space.yaml

```diff
@@ -0,0 +1,20 @@
+openapi: 3.0.3
+info:
+  version: "1.0.0"
+  title: Invalid API
+security:
+  - "Basic Access Authentication": ["read", "write"]
+paths:
+  /:
+    post:
+      responses:
+        200:
+          description: OK
+components:
+  securitySchemes:
+    "Basic Access Authentication":    # <---- component names shouldn't have a space in them
```

Comment:
Per the spec: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#components-object

> All the fixed fields declared above are objects that MUST use keys that match the regular expression: ^[a-zA-Z0-9\.\-_]+$.